### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pjflux2001**
1. Added links to helpful commands for helm, kubectl and minikube
2. Fixed typo
3. Made kubectl logs step as dynamic - cause same name won't be used on every run

### CODE REVIEW

**Comment by koolgax99** [comment]
Hey, your grep only works for unix type bashes. Sadly, it wont work for Windows.

**Comment by pjflux2001** [comment]
made changes - added support for windows too

File: README.md [hunk 1]
```diff
@@ -99,13 +103,19 @@ _NOTE: Replace the pod IDs with the ones from your instance, they change every t
 
       kubectl apply -f ftsdb.yaml
       
-      kubectl logs -f fts-mysql-db7988d96-gn6dw
+      kubectl logs -f $(kubectl get pods -o NAME | grep fts-mysql | cut -d '/' -f 2)
+      
+for windows:
```
Comment (mlassnig) [inline README.md:108]: Capital f, capital w :-)

File: README.md [hunk 2]
```diff
@@ -99,13 +103,19 @@ _NOTE: Replace the pod IDs with the ones from your instance, they change every t
...
+      kubectl logs -f $(kubectl get pods -o name | findstr /c:"fts-mysql" | sed "s/^pod\///")
 
 * Install FTS, once the FTS database container is up and running:
 
       kubectl apply -f fts.yaml
       
-      kubectl logs -f fts-server-7cb5d7c789-scg6c
+      kubectl logs -f $(kubectl get pods -o NAME | grep fts-server | cut -d '/' -f 2)
+      
+for windows:
```
Comment (mlassnig) [inline README.md:117]: Capital f, capital w :-)

File: README.md [hunk 3]
```diff
@@ -50,14 +50,18 @@
 * Shut down minikube:
 
       minikube stop
+* For more useful commands:
```
Comment (mlassnig) [inline README.md:53]: I think this needs an empty line for better readability.

**Pull request merged**